tap_shiphero/client.py

```diff
@@ -63,7 +67,21 @@ def request(self, method, path, **kwargs):
 
         response.raise_for_status()
 
-        return response.json()
+        try:
+            return response.json()
+        except:
+            LOGGER.info('----------------------------------------')
+            # I want as much information here as possible, but I really
+            # only care about the `content-type` here
+            LOGGER.info('response.headers: ', response.headers)
```

Comment:
Only log content-type if you're going to log. response.headers may have the API key in it.